portfolio/src/main/webapp/comments.js

```diff
@@ -12,25 +12,71 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * An object to manage the query string of a URL
+ * @const {URLSearchParams}
+ */
+const GET_PARAMS = new URLSearchParams ({
+  quantity: '5',
+  dateOrder: 'true'
+});
+
 /**
  * Fetches comments from the server and adds it to the DOM.
  */
 function getComments() {
-  fetch('/comments').then(response => response.json()).then((comments) => {
-    const commentsContainer = document.getElementById('comments-container');
+  const url = '/comments?' + GET_PARAMS.toString();
 
-    commentsContainer.innerHTML = '';
-    for (const comment of comments) {
-      commentsContainer.appendChild(createListElement(comment));
-    }
+  fetch(url).then(response => response.json()).then((comments) => {
```

Comment:
With the previous tip you can do something like this:
fetch(`/comments?${GET_PARAMS.toString()}`).then(response => response.json()).then((comments) => {